src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -697,7 +696,7 @@ def exec_line(
         """
         self._node_concurrency = node_concurrency
         inputs_with_default_value = FlowExecutor._apply_default_value_for_input(self._flow.inputs, inputs)
-        inputs = self._process_images_from_inputs(self._flow.inputs, inputs_with_default_value)
+        inputs = load_multimedia_data(self._flow.inputs, inputs_with_default_value, self._working_dir)
```

Comment:
What if customer's dict with image value is passed in as node.input?

Reply:
Offline synced, will fix in new PR.